client/src/app/users/user-list.component.ts

```diff
@@ -39,7 +39,23 @@ export class UserListComponent implements OnInit {
       this.serverFilteredUsers = returnedUsers;
       this.updateFilter();
     }, err => {
-      console.log(err);
+      // If there was an error getting the users, display
+      // a message.
+      this.snackBar.open(
+        'Problem contacting the server – try again',
+        'OK',
```

Comment:
I'm not sure we need this button here and I'm pretty sure it doesn't actually do anything.

Reply:
Actually it _does_ do something – it dismisses the snack bar. Otherwise you have to wait the 3 seconds for it to disappear. And clicking elsewhere on the snack bar doesn't dismiss it. I'm inclined to keep it so the user feels that they can do _something_.